src/chain/addresses/kovan-mcd.json

```diff
@@ -1,6 +1,6 @@
 {
-  "CHIEF": "0xcba4147decf3c7c2544b99a3c8510050f85b99a1",
-  "VOTE_PROXY_FACTORY": "0xdab69a749194dc39c3c2bbabc29e87ebb4f07215",
-  "GOV": "0xb54d411a3cf9cd2301917d558c05c174612da367",
-  "IOU": "0xec1412c71aa154c02de368f4921a636fa441b973"
+  "CHIEF": "0xe5d22b1887cac1e3f84c5bbefb7c25526e6a0c37",
```

Comment:
might leave the address changes in this file out of this PR. or are they necessary for the functionality here?

Reply:
No, they are unused, I think those commits are included as I pulled from develop directly, instead of pulling develop -> GOV-202 -> here. I'll see if I can clean it up